### DISCUSSION TITLE
Good day! How do I change 404 not found page? I like to change its style.

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#19289] [Questions]
Good day! How do I change 404 not found page? I like to change its style.

what I am trying to achieave is something like this:

![image](https://user-images.githubusercontent.com/31676163/221405076-d2897311-614d-4990-8ea3-a2c1f9fcf82a.png)

**Comment by videmann** [comment]
You can create /pages/404.vue and use navigateTo('/404') or create error.vue on root of project to handle all statuses by one page and use throw createError({ statusCode: 404, message: 'page not found' }), for example. use fatal key if you need to show error page on client. You welcome!

**Reply by JenuelDev** [reply]
I did those but still, nothing happened, it still shows Document not found.

to give an idea of what I got. I have 3 pages only.

1. `/` - basically the main index page
2. `/about`
3. `/contact`

when I go to a page or route that does not exit I get `<div>Document not found</div>`

**Comment by TheAlexLichter** [comment]
Check out the [`error.vue` documentation](https://nuxt.com/docs/getting-started/error-handling#rendering-an-error-page). That should be what you want ☺️

**Reply by JenuelDev** [reply]
I aactually did that,, but still its not rendering the error.vue,, but it shows this
![image](https://user-images.githubusercontent.com/31676163/221404975-6b64f542-a208-40de-a49d-88c3c511b1d3.png)

**Reply by TheAlexLichter** [reply]
Could you provide a reproduction? Just noticed you use a custom router instead of file-based. That could change some thing 🙈

**Reply by JenuelDev** [reply]
it is because I have a default layout?

**Reply by TheAlexLichter** [reply]
Default layout should be fine. Provide what you have [as reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction) and I'll have a closer look 👍
